support/cas-server-support-cassandra-ticket-registry/build.gradle

```diff
@@ -0,0 +1,24 @@
+
+description = "Apereo CAS Cassandra ticket registry"
+dependencies {
+    compile project(":core:cas-server-core")
+    compile project(":core:cas-server-core-tickets")
+    compile project(":core:cas-server-core-configuration")
+    compile project(":core:cas-server-core-logout")
+
+    compile libraries.cassandra
+    compile libraries.spring
```

Comment:
The only one you really need here is cassandra. Everything else is provided for you.